src/com/pentalog/pentastagiu/homework4/MapOfEmployees/Application.java

```diff
@@ -0,0 +1,31 @@
+package com.pentalog.pentastagiu.homework4.MapOfEmployees;
+
+import com.sun.deploy.security.SelectableSecurityManager;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Application {
+    public static void main(String[] args) {
+        Map<Integer, List<String>> Employees = new HashMap<>();
+        Employees.put(30, Arrays.asList("Andrei","Flavius"));
+        Employees.put(35,Collections.singletonList("Marius"));
+        int newKey=20;
+        if(Employees.containsKey(newKey)) {
```

Comment:
Nu are rost sa folosesti un if gol pentru a merge pe bransa else. Daca vrei sa faci ceva in caz ca conditia e falsa poti folosi `!` asa:  `if(!Employees.containsKey(newKey)) {***code here***}`